pkg/sql/execinfra/metrics.go

```diff
@@ -73,6 +75,24 @@ var (
 		Measurement: "Nanoseconds",
 		Unit:        metric.Unit_NANOSECONDS,
 	}
+	metaCumulativeSQLCPUNanos = metric.Metadata{
+		Name:        "sql.distsql.cumulative_sql_cpu_nanos",
+		Help:        "Cumulative CPU time spent executing SQL operations across all queries (in nanoseconds)",
+		Measurement: "Nanoseconds",
+		Unit:        metric.Unit_NANOSECONDS,
+		Essential:   true,
+		Category:    metric.Metadata_SQL,
+		HowToUse:    `Monitor this metric to track overall SQL CPU consumption across the cluster.`,
+	}
+
+	metaSQLCPUStatementsCount = metric.Metadata{
+		Name:        "sql.distsql.cpu_sql_statements.count",
+		Help:        "Number of SQL statements for which CPU time was recorded",
+		Measurement: "Statements",
```

Comment:
For consistency, let's rename this to `"Queries"` so that it matches the other metrics in this file.